### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by pomber** [code-hike/codehike#84] [Announcements]
These are the next steps for Code Hike:

- [x] Publish the [new website](https://code-hike.vercel.app/) _(first week of December 2021)_
Including demos for all the main features
- [x] Publish a version for early adopters and some minimal docs/demos _(Dec 2021 / Jan 2022)_
The goal is to gather feedback on the MDX API, and identify what features are missing
- [ ] 🚀 `v0.3.0`: stable codeblocks + docs 
- [ ] 🚀 `v0.4.0`: stable annotations + docs 
- [ ] 🚀 `v0.5.0`: stable `<CH.Code/>` + docs

**Comment by Aiyush-G** [comment]
Wow, the [new website](https://code-hike.vercel.app/) looks brilliant! The UI of each of the sections is slick and the overall aesthetic quality is immense!

**Reply by pomber** [reply]
Thank you, there are a few things missing, but all the heavy work is done.

**Reply by Aiyush-G** [reply]
If I may, how did you go about the design (of the interface) / which UI framework(s) were used, if any?

**Reply by pomber** [reply]
It's Tailwind and Next.js. The code is here https://github.com/pomber/code-hike-site

**Comment by paolotiu** [comment]
Looks dope, I'm so excited for this!
